frontend/src/concepts/pipelines/content/experiment/ManageExperimentModal.tsx

```diff
@@ -15,11 +15,10 @@ import useCreateExperimentData from '~/concepts/pipelines/content/experiment/use
 import { ExperimentKFv2 } from '~/concepts/pipelines/kfTypes';
 
 type ManageExperimentModalProps = {
-  isOpen: boolean;
   onClose: (experiment?: ExperimentKFv2) => void;
 };
 
-const ManageExperimentModal: React.FC<ManageExperimentModalProps> = ({ isOpen, onClose }) => {
```

Comment:
🤔 why did you change this?

Reply:
I think it was based on a conversation I had a while ago with @christianvogt because we wanted to refresh the modal state every time we open/close it.

Reply:
I recall a conversation around using `key` attribute to force re-rendering a component. I don't believe it was related to this scenario.

I also recall a conversation about how to tend to do a lot of "work" while models are closed because of the pattern we've adopted by rendering modal wrapping components such as this which perform the work even though the modal itself is not yet open. Perhaps that's what trigger this? I honestly I don't remember anything specific to this component.

Reply:
We had a conversation sometime back while peer programming to solve some issues. I don't recall the specifics. There wasn't any direction given about sweeping changes.
Again, changes to this modal i don't recall specifically discussing.

Reply:
looks like this was reverted. so should be resolved